src/main/java/uk/gov/pay/adminusers/model/InviteServiceRequest.java

```diff
@@ -1,63 +1,55 @@
 package uk.gov.pay.adminusers.model;
 
-import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import io.swagger.v3.oas.annotations.media.Schema;
+import uk.gov.pay.adminusers.validations.ValidEmail;
+import uk.gov.pay.adminusers.validations.ValidTelephoneNumber;
 
-import java.util.Optional;
+import javax.validation.constraints.NotEmpty;
 
-import static uk.gov.pay.adminusers.app.util.RandomIdGenerator.randomUuid;
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class InviteServiceRequest {
+    
+    private String password;
 
-public class InviteServiceRequest extends InviteRequest {
-    public static final String FIELD_PASSWORD = "password";
-    public static final String FIELD_TELEPHONE_NUMBER = "telephone_number";
-    private static final String DEFAULT_ROLE_NAME = "admin";
+    @NotEmpty
+    @ValidEmail
```

Comment:
I _think_ there's a javax/ hibernate validator for email which might be able to do some of the custom validator lifting. 

I'm not sure if it validates to the same strictness as the apache commons validator but may be good enough given that we check it's a valid public sector email on top of this constraint.

```java
@NotEmpty
@Email(message = "must be a valid email address")
protected String email;
```

Reply:
Ah nice, it would save on some code using that. The intention here is probably just to ensure that we don't have non-emails coming through. We also do validation on the frontend so there shouldn't be invalid emails coming through anyway.